src/cdi_analytic/Analytic_Models.cc

```diff
@@ -429,6 +429,33 @@ Constant_Analytic_EICoupling_Model::get_parameters() const {
   return sf_double(1, ei_coupling);
 }
 
+//===========================================================================//
+// ANALYTIC_KP_ALPHA_ELOSS_MODEL MEMBER DEFINITIONS
+//===========================================================================//
+
+/*! \brief Calculate the eloss in units of shk^-1; T given in keV, rho in g/cc,
+ *         v0 in cm/shk
+ *
+ * \param T material temperature field in keV
+ * \param rho material density in g/cm^3
+ * \param v incident particle speed in cm/shk
+ * \return eloss (vector of time coefficients) in shk^-1
+ * 
+ */
+double Analytic_KP_Alpha_Eloss_Model::calculate_eloss(double T, double rho,
+                                                      double v) const {
+  double range = 0.03 * T * (1.0 - 0.24 * std::log(1.0 + T)) *
```

Comment:
* Add `Require()` entries? especially to rho, T > 0.
* Why 0.37, 0.03, 0.24, etc?  Should these be adjustable parameters?